libs/filabridge/include/private/filament/EngineEnums.h

```diff
@@ -71,6 +72,23 @@ enum class ReservedSpecializationConstants : uint8_t {
     CONFIG_STEREO_EYE_COUNT = 8, // don't change (hardcoded in ShaderCompilerService.cpp)
 };
 
+
+
+// Note that the following enum/arrays should be ordered so that the ids correspond to indices in
+// the two vectors.
+enum class PushConstantIds {
+    MORPHING_BUFFER_OFFSET = 0,
+};
+
+using PushConstantType = backend::ConstantType;
+
+const utils::FixedCapacityVector<char const*> PUSH_CONSTANT_NAMES = {
+    "morphingBufferOffset",
+};
+const utils::FixedCapacityVector<PushConstantType> PUSH_CONSTANT_TYPES = {
+    PushConstantType::INT,
+};
```

Comment:
I would prefer a single vector of pair<name, type>. And in fact maybe we need two such vector, one for each stage.

Reply:
done